sdk/Eversign/Signer.php

```diff
@@ -235,7 +235,7 @@ public function setSigned($signed) {
         $this->signed = !!$signed;
     }
 
-    public function setSigned_timestamp($signed_timestamp) {
+    public function setSignedTimestamp($signed_timestamp) {
```

Comment:
getSignedTimestamp would be more consistent than `getSigned_timestamp`.
However, this is public function and this was released since a long time.

Meaning, changing it right now would introduce a breaking change. It might break other integrations. 
On the other hand this is not a big enough change to make a major release.

So, in short, we can not change this now. Please revert this part of the change.

Reply:
Ok I understand, I added a deprecated function with the "_" to not BC the current.